content/responses/sign_request.yml

```diff
@@ -0,0 +1,101 @@
+---
+title: Sign Request
+
+type: object
+
+x-box-resource-id: sign_requests
+x-box-tag: sign_requests
+
+description: |-
+  A Sign Request Object
+
+allOf:
```

Comment:
We won't be able to use allOf in the Gundam schema, so it may make sense to match the two by expanding this tag?

Reply:
I think it's more important to minimize duplicate code - we can work with the Gundam team to request support be added for allOf in those repositories